compiler/luci-interpreter/src/kernels/Concatenation.test.cpp

```diff
@@ -159,6 +168,23 @@ TEST(ConcatenationTest, Unsupported_Configure_Type_NEG)
   ConcatenationParams params{};
 
   params.axis = -1;
+  params.activation = luci::FusedActFunc::NONE;
+
+  Concatenation kernel({&input1_tensor, &input2_tensor}, &output_tensor, params);
+  EXPECT_ANY_THROW(kernel.configure());
+}
+
+TEST(ConcatenationTest, With_Fused_Activation_NEG)
```

Comment:
Does this test needs to be removed when fused activation works?

Reply:
Yes. At that time, this test should be removed.

Reply:
Ah, then please add a comment about this...